python/lsst/ts/m2gui/control_tabs.py

```diff
@@ -121,9 +125,13 @@ async def _callback_show(self, item):
         """
 
         control_tab = self._get_control_tab(item.text())
-        control_tab.show()
 
-    def get_tab(self, name):
+        if control_tab is not None:
+            control_tab.show()
+
+    def get_tab(
+        self, name: str
+    ) -> typing.Tuple[QListWidgetItem | None, TabDefault | None]:
```

Comment:
I think you can use `tuple` instead of `typing.Tuple`, same argument as the `list`...

Reply:
Fixed in the commit: [b8ec396](https://github.com/lsst-ts/ts_m2gui/pull/35/commits/b8ec3962d7117af883c72fd25835442da89caf3d). Thanks!